src/Kodify/BlogBundle/Tests/Controller/CommentsControllerTest.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace Kodify\BlogBundle\Tests\Controller;
+
+use Kodify\BlogBundle\Entity\Comment;
+use Kodify\BlogBundle\Entity\Post;
+use Kodify\BlogBundle\Entity\Author;
+use Kodify\BlogBundle\Tests\BaseFunctionalTest;
+
+class CommentsControllerTest extends BaseFunctionalTest
+{
+
+    protected function createComments($count)
+    {
+        $author = new Author();
+        $author->setName('Author');
+        $this->entityManager()->persist($author);
+        $this->entityManager()->flush();
+
+        $post = new Post();
+        $post->setTitle('Title');
+        $post->setContent('Content');
+        $post->setAuthor($author);
+        $this->entityManager()->persist($post);
+        $this->entityManager()->flush();
+
+        for ($i = 0; $i < $count; ++$i)
+        {
+            $comment = new Comment();
+            $comment->setContent('Comment content ' . $i);
+            $comment->setAuthor($author);
+            $comment->setPost($post);
+            $comment->setCreatedAt( new \DateTime() );
+            $comment->setUpdatedAt( new \DateTime() );
```

Comment:
You're absolutely right. I started writing it my way, since we do not have any coding standards at my current job, and as I kept going I started to mimic the style of the project. As a result, there was a bit of a messy combination.